### ISSUE TITLE
Tri-State Checkbox Example: clarify usage of aria-controls

### ISSUE THREAD

**Issue by jscholes** [w3c/aria#1352]
In the [Tri-State Checkbox Example](https://w3c.github.io/aria-practices/examples/checkbox/checkbox-2/checkbox-2.html), `aria-controls` is applied to the "All condiments" checkbox, with a value referencing all four IDs of the individual condiment two-state checkboxes.  This is programmatically correct, but:

* `aria-controls` usage isn't mentioned anywhere within the [Checkbox design pattern](https://w3c.github.io/aria-practices/#checkbox); and
* as covered in https://github.com/w3c/aria/issues/995, there is still no consensus about how `aria-controls` should be surfaced by assistive technology.  Indeed, the previous surfacing by JAWS (now silenced by default) didn't take multiple IDREFs into account.

Given the above, should `aria-controls` usage be added to the "WAI-ARIA Roles, States, and Properties" section of the Checkbox design pattern?  And for the purposes of ARIA-AT testing (ref: https://github.com/w3c/aria-at/issues/322), my belief is that no assertions can be included as of the time of filing, because there is no meaningful understanding of how to surface the information.

**Comment by jscholes** [comment]
Accidentally filed here due to browser autocomplete mistake.  Apologies; actual issue is: https://github.com/w3c/aria-practices/issues/1599
